internal/scorecard/plugins/resource_handler.go

```diff
@@ -248,7 +248,7 @@ func createKubeconfigSecret(namespace string) error {
 		return err
 	}
 	addResourceCleanup(kubeconfigSecret, types.NamespacedName{Namespace: kubeconfigSecret.GetNamespace(),
-		Name: kubeconfigSecret.GetName()})
+		Name: kubeconfigSecret.GetName()}, 60)
```

Comment:
should not here be the initTimeout as well?

Reply:
good catch, fixed, also updated the CHANGELOG and docs with this latest commit.